script/SignFromJson.s.sol

```diff
@@ -35,6 +35,5 @@ contract SignFromJson is MultisigBuilder, JsonTxBuilderBase {
     {
         accesses; // Silences compiler warnings.
         simPayload;
-        console.log("\x1b[1;33mWARNING:\x1b[0m _postCheck not implemented");
-    }
```

Comment:
Looks like you accidentally deleted the `}` which is causing a compiler failure.

Reply:
Oops, I accidentally forgot it, that's why CI is crashing, I updated it.